obp-api/src/main/scala/code/actorsystem/ObpActorInit.scala

```diff
@@ -20,38 +21,20 @@ trait ObpActorInit extends MdcLoggable{
   val TIMEOUT = (ACTOR_TIMEOUT seconds)
   implicit val timeout = Timeout(ACTOR_TIMEOUT * (1000 milliseconds))
 
-  /**
```

Comment:
totally wrong use of generic and match case, and asInstanceOf.
so just remove it, and original caller switch to call extractFutureToBox method